### PR TITLE
Add AMP_BIND, AMP_ACCESS, AMP_LIST, AMP_MUSTACHE scripts to amp-boilerplate exports

### PR DESCRIPTION
**Pull request by sareh**
Part of https://github.com/bbc/simorgh/issues/4586

**Overall change:** Adds three new AMP scripts so they're available to be used for containers within Simorgh. The AMP Access, Bind, List and Mustache scripts will be needed for data fetching at the container level within AMP pages. 

**Code changes:**

- Add scripts from 
https://amp.dev/documentation/components/amp-access/ 
https://amp.dev/documentation/components/amp-bind/ 
https://amp.dev/documentation/components/amp-list/ 
https://amp.dev/documentation/components/amp-mustache/
- Added unit tests (snapshots) for all amp-boilerplate exports. 

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by thekp — APPROVED** [review]
💯

**Review by EinsteinNjoroge — APPROVED** [review]
:+1:

File: packages/utilities/psammead-assets/src/amp-boilerplate.jsx [hunk 1]
```diff
@@ -21,10 +36,24 @@ export const AMP_CONSENT_JS = (
     src="https://cdn.ampproject.org/v0/amp-consent-0.1.js"
   />
 );
-export const AMP_ANALYTICS_JS = (
```
Comment (sareh) [inline packages/utilities/psammead-assets/src/amp-boilerplate.jsx:24]: This isn't removed, just re-ordered to have the JS exports listed alphabetically.

**Review by j-pendlebury — APPROVED** [review]
LGTM

**Pull request merged**